spec/exchanges/coins_markets/integration/market_spec.rb

```diff
@@ -14,20 +14,19 @@
   end
 
   it 'fetch ticker' do
-    dog_xgtc_pair = Cryptoexchange::Models::MarketPair.new(base: 'DOG', target: 'XGTC', market: 'coins_markets')
-    ticker = client.ticker(dog_xgtc_pair)
+    pair = Cryptoexchange::Models::MarketPair.new(base: 'BTC', target: 'DOG', market: 'coins_markets')
```

Comment:
@rusbal After the target/base flip, i would imagine that for this example
base should be DOG, target should be BTC